docs/src/main/sphinx/installation/cli.rst

```diff
@@ -25,6 +25,9 @@ make it executable with ``chmod +x``, then run it:
 
 Run the CLI with the ``--help`` option to see the available options.
 
+Documentation on the HTTP protocol between the Trino CLI and the Trino
```

Comment:
Maybe add same thing to the JDBD driver? 

And also to the other repos (go client and python client) once published.

Reply:
Added to the JDBC driver doc.

Reply:
Reword to use descriptive link text

```
The CLI uses the HTTP protocol and the :doc:`Trino client REST API </develop/client-protocol>` to communicate with Trino.
```

And similar in JDBC

Reply:
Changed.